src/fontra/views/editor/editor.js

```diff
@@ -1520,7 +1383,7 @@ export class EditorController {
   async toggleUserSettings(onOff) {
```

Comment:
Do you mean to address the toggle API as a separate task? I'd really like to get rid of this in editor.js.